src/helpers/dateHelpers.ts

```diff
@@ -0,0 +1,39 @@
+export function getCurrentDate(): string {
+  const date = Date(),
+    d = new Date(date),
+    year = d.getFullYear();
+
+  let month = '' + (d.getMonth() + 1),
+    day = '' + d.getDate();
+
+  if (month.length < 2) month = '0' + month;
+  if (day.length < 2) day = '0' + day;
+
+  return [year, month, day].join('-');
+}
+
+export function dateJoinedMessageFileter(yearDifference, monthDifference, dayDifference): string {
+  if (yearDifference == 1) {
+    return `Joined ${yearDifference} year ago`;
+  } else if (yearDifference > 1) {
+    return `Joined ${yearDifference} years ago`;
+  } else if (monthDifference == 1) {
+    return `Joined ${monthDifference} month ago`;
+  } else if (monthDifference > 1) {
+    return `Joined ${monthDifference} months ago`;
+  } else if (dayDifference == 1) {
+    return `joined ${dayDifference} day ago`;
```

Comment:
Can you capitalize the `j` here 
```suggestion
    return `Joined ${dayDifference} day ago`;
```

Reply:
yea, right away

Reply:
Done